lib/internal/streams/pipeline.js

```diff
@@ -240,7 +240,9 @@ function pipeline(...streams) {
         // always returns a stream which can be further
         // composed through `.pipe(stream)`.
 
-        const pt = new PassThrough();
+        const pt = new PassThrough({
+          objectMode: true
```

Comment:
Is it ok to always use object mode? Doesn't this change how buffering work? It counts 16 "objects" instead of 16 KB.

Reply:
That’s how it works with generators and iterators. See Readable.from I think the same thing was discussed there. I can’t find it at the moment. @mcollina maybe you have some insight?

Reply:
Either way I don’t think we actually want buffering here. I am tempted to even set the watermark to 1.

Reply:
It is technically possible to write 16 buffer of 1 GB each with no backpressure right?

Reply:
Yes, not sure what you are getting at though? The purpose here is just to create an APO proxy to match the expected return value of pipeline.

Reply:
Ok.